### DISCUSSION TITLE
Commercial licence

### DISCUSSION THREAD

**Discussion by HametAl** [kinnala/scikit-fem#1045] [Q&A]
Hi Scikit team

I am wondering if there is any commercial licence for scikit? Can I use it in my closed source commercial software?

Thanks and regards

**Comment by kinnala** [comment]
There are no separate commercial licences or support available. However, the default licence is here and allows certain type of commercial activity: https://github.com/kinnala/scikit-fem/blob/master/LICENSE

Usually commerical software using BSD or MIT licenced source code includes a listing of the copyright notices as part of their manual.

**Reply by HametAl** [reply]
> There are no separate commercial licences or support available. However, the default licence is here and allows certain type of commercial activity: https://github.com/kinnala/scikit-fem/blob/master/LICENSE
> 
> Usually commerical software using BSD or MIT licenced source code includes a listing of the copyright notices as part of their manual.

Thanks for your reply. So, my understanding from the license description is that I can use scikit-fem libraries in my commercial software without any restriction as long as I reproduce the copyright notice and the conditions mentioned in the license file?

**Reply by kinnala** [reply]
Yes. Of course if you have any bug reports or suggestions or even contributions, you can post them here and any volunteers may help you out.

**Reply by HametAl** [reply]
Great, thanks!
